runtime/exports.t4

```diff
@@ -177,23 +177,23 @@
 
 		#region metadata/profiler.h
 
-		new Export ("void", "mono_profiler_set_events",
-			"MonoProfileFlags", "events"
+		new Export ("MonoProfilerHandle", "mono_profiler_install",
```

Comment:
Assuming this does _not_ affect existing non-profiling use cases, I believe it is fine to say "if you want to profile a XM app with XM X you need at least Mono Y.

We require matching mono/XM for classic and somewhat matching VSfM/XM, so it isn't unheard of. As long as this won't change often and creating a nightmare later i'm 👍